### DISCUSSION TITLE
How to add receiver.dat file in SeisSol parameters.par ?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#831] [Q&A] [answered]
I recently read an [article](https://mediatum.ub.tum.de/doc/1612172/5iit0j3aedqsc79h7mv6olowr.TimeReversalOfSeismicWaves_PhilippWendland.pdf), I find SeisSol support `Time-Reversal`, I want to test it, so I set a series of receivers and get  `loh3-receiver-00001-00161.dat` 、`loh3-receiver-00002-00161.dat  ` and so on, How can I add receiver.dat file in SeisSol parameters.par ? I can not find how to add in the parameters.par in your [documentation](https://seissol.readthedocs.io/en/latest/parameter-file.html) .

**Comment by sebwolf-de** [comment, marked as answer]
Hi, the time-reversal implementation is experimental and never made it into the production stage. You can find the code under: https://github.com/SeisSol/SeisSol/tree/sebastian/time-machine-prototype
